### ISSUE TITLE
List files inside a directory and its subdirectory

### ISSUE THREAD

**Issue by mainawycliffe** [bitfield/script#35]
We should introduce a method that will list all files in a directory and its subdirectories and return a pipe just like the ListFiles Method. Probably the method should be able to filter hidden files using a `showHidden` argument. I am still yet to come up with a decent name for the method, I 
am open to suggestions. 

```go
func FindFiles(path string, showHidden bool) *Pipe {
  // content
}
```

**Comment by bitfield** [comment]
Sounds good, @mainawycliffe! Let's punt the question of hidden files for the moment and try to come up with the simplest thing that could possibly work.

`FindFiles` sounds like a good name. Given the following directory tree:

```
foo/
-- file1.txt
-- file2.txt
-- bar/
----file3.txt
```

I guess the output of FindFiles should be:

```go
foo/file1.txt
foo/file2.txt
foo/bar/file3.txt
```

What do you think?

**Comment by mainawycliffe** [comment]
> Sounds good, @mainawycliffe! Let's punt the question of hidden files for the moment and try to come up with the simplest thing that could possibly work.

Yes, we should keep things as simple as possible. 

> FindFiles sounds like a good name. Given the following directory tree:
> 
> foo/
> -- file1.txt
> -- file2.txt
> -- bar/
> ----file3.txt
> I guess the output of FindFiles should be:
> 
> foo/file1.txt
> foo/file2.txt
> foo/bar/file3.txt
> What do you think?

Yeah, the output should be exactly like that.

**Comment by bitfield** [comment]
Fixed by #36!
